kernel/src/main/java/org/kframework/compile/checks/CheckTokens.java

```diff
@@ -0,0 +1,43 @@
+package org.kframework.compile.checks;
+
+import com.google.common.collect.ImmutableSet;
+import org.kframework.definition.Module;
+import org.kframework.definition.Production;
+import org.kframework.definition.Sentence;
+import org.kframework.utils.errorsystem.KEMException;
+
+import java.util.Set;
+
+public class CheckTokens {
+    private final Set<KEMException> errors;
+    private final Module m;
+    private static final ImmutableSet<String> ignoredSortNames = ImmutableSet.of("Bag", "IOFile", "IOInt", "IOString", "K", "KBott", "KItem", "KLabel", "KList", "Sort");
```

Comment:
Why `Sort`? Where is that coming from?
Not sure about some others also. `KItem` for example.
I just noticed that in the original issue, I complained about the LLVM and Haskell backend. But Java is a bit different. Probably we don't need to run this check there.
Could you check all these, please?

Reply:
I came up with these by running kompile on the tokenCheck.k file and looking at the errors that were generated without ignoring special sort names and `Sort` was one of them

Reply:
You should check all of them. They might indicate something else is wrong.

Reply:
oh strange... Now I'm getting that I only need to ignore "KBott" and "KLabel"

Reply:
I think this was written when I was still working on solidifying the logic. I removed all but KBott and KLabel and I'm rerunning the testsuite locally. I'll push another commit addressing this and the other feedback you have. Thanks!